Brisk/Controllers/RadarViewController.swift

```diff
@@ -60,6 +61,9 @@ final class RadarViewController: ViewController {
         self.classificationPopUp.setItems(titles: Classification.All.map { $0.name })
         self.reproducibilityPopUp.setItems(titles: Reproducibility.All.map { $0.name })
         self.productPopUp.set(items: Product.All, getTitle: { $0.name }, getGroup: { $0.category })
+        self.attachmentDroppableView.droppedAttachment = { [weak self] attachments in
+            self?.attachments = [attachments]
```

Comment:
Right now since we're not supporting multiple attachments this is a replacement. Eventually it will definitely be an append but it gets weird with the current UI since you don't know what you have attached, and removing removes all of them.